custom_model_runner/datarobot_drum/drum/common.py

```diff
@@ -44,8 +48,23 @@ def verbose_stdout(verbose):
         sys.stdout = old_target
 
 
+def request_id_filter(record: logging.LogRecord):
+    """
+    A filter which injects context-specific information into logs and ensures
+    that only information for a specific webapp is included in its log
+    """
+    request_id = ctx_request_id.get(None)
+    record.context_data = {"request_id": request_id} if request_id else ""
+    return True
+
+
 def config_logging():
-    logging.basicConfig(format="%(asctime)-15s %(levelname)s %(name)s:  %(message)s")
+    stream_handler = logging.StreamHandler()
+    stream_handler.addFilter(request_id_filter)
+    logging.basicConfig(
+        format="%(asctime)-15s %(levelname)s %(name)s:  %(message)s %(context_data)s",
```

Comment:
`request_id` is part of the `context_data`. It's formatted at the end of log messages so that `request_id` didn't shift the message when displayed on UI (Runtime Logs and MLOps Events).